.github/workflows/run_vi_analyzer.yml

```diff
@@ -7,14 +7,39 @@ on:
 jobs:
   run_vi_analyzer:
     name: Run VI Analyzer
-    runs-on: self-hosted
+    runs-on: 
+      - self-hosted
+      - windows
+      - x64
+      - rdss-measlinkbot-win-10-lv64
+    strategy:
+      matrix:
+        version: ["2021", "2022", "2023"]
```

Comment:
Let's just do one version for now. I know we talked about running this for all versions, but I can't see any benefit to doing that.

Reply:
Updated